src/lib/gui/shaky_cam.h

```diff
@@ -59,10 +65,10 @@ class shaky_cam
 				offsetX += (*_shakePos).first;
 				offsetY += (*_shakePos).second;
 			}
-			++_shakePos;
 		}
 
 		img.copyTo(_frame(cv::Rect(offsetX, offsetY, img.cols, img.rows)));
+		cv::cvtColor(_frame, _frame, cv::COLOR_BGR2RGB);
```

Comment:
It's not really clear where this should live, but for now it's here. We spit out RGB images, opencv wants BGRs, at some point before we render to screen (or to file) with opencv we need to do the conversion.